homeassistant/components/cert_expiry/sensor.py

```diff
@@ -82,6 +82,10 @@ def available(self):
         """Icon to use in the frontend, if any."""
         return self._available
 
+    async def async_added_to_hass(self):
+        """Once the entity is added we should update to get the initial data loaded."""
+        self.update()
```

Comment:
To keep the old behavior we should listen for home assistant start event and do an update then.